Sources/SourceKitLSP/Clang/ClangLanguageService.swift

```diff
@@ -622,7 +622,9 @@ extension ClangLanguageService {
     return try await forwardRequestToClangd(req)
   }
 
-  func openGeneratedInterface(_ request: OpenGeneratedInterfaceRequest) async throws -> GeneratedInterfaceDetails? {
+  func openGeneratedInterface(document: DocumentURI, moduleName: String, groupName: String?, symbolUSR symbol: String?)
+    async throws -> GeneratedInterfaceDetails?
+  {
```

Comment:
Nitpick: I prefer to wrap the function on the arguments instead of the return type

```suggestion
  func openGeneratedInterface(
    document: DocumentURI, 
    moduleName: String, 
    groupName: String?, 
    symbolUSR symbol: String?
  ) async throws -> GeneratedInterfaceDetails? {
```